torch/onnx/_internal/jit_utils.py

```diff
@@ -381,4 +381,4 @@ def parse_node_kind(kind: str) -> Tuple[str, str]:
 @_beartype.beartype
 def is_custom_domain(domain: str) -> bool:
     """Check if the domain is official."""
-    return domain not in {"onnx", "aten", "prim"}
+    return domain not in {"onnx", "aten", "prim", ""}
```

Comment:
Does `""` happen?

Reply:
Yes, cause I use onnx model proto instead, and there might be identity without domain.

Reply:
Would it be possible to normalize the name before calling this function? I think it would make the code more robust to have "always have a domain" as an invariant

Reply:
I guess I can check if the node proto has domain field or not, but another question is that is "" one of the domain? I remember we accept that in some cases, and take them as aten/onnx for default.

Reply:
Open a new PR to add onnx_proto_utils.py for internal utils function in utils.py

Reply:
#88376

Reply:
Just found another issue with this approach. The domain startswith `org.pytorch` is also official... kind of lost in the pattern we have in here...